src/__generated__/About_artist.graphql.ts

```diff
@@ -0,0 +1,106 @@
+/* tslint:disable */
```

Comment:
Ah, shame, guess the gitattributes has to be merged before we get to use it in PRs

Reply:
What do you mean? Is it not working for you?

Reply:
They're flattened, yep, but I can't tell if that's because the PR is too long because all of the in app source code changes are also flattened after it.

Reply:
Ah I see. I’m pretty sure that some of these artefacts were expanded by default yesterday.